geoip2/records.py

```diff
@@ -629,6 +629,14 @@ class Traits(Record):
 
       :type: unicode
 
+    .. attribute:: user_count
+
+      The estimated number of users sharing the IP/network during the past 24
+      hours. For IPv4, the count is for the individual IP. For IPv6, the count
+      is for the /64 network.
```

Comment:
Can you mention that this is only available for the GeoIP2 Precision Insights service? This class is shared across all the web services and location databases.